### DISCUSSION TITLE
Links points to wrong URL on "Dojo for WooCommerce" landing page

### DISCUSSION THREAD

**Discussion by malaporpism** [dojo-engineering/feedback#18] [Docs]
All six child links "Introduction" "Install" "Configure" "Release Notes" "Enable Dojo Checkout" "Need a Hand" on https://docs.dojo.tech/plugins/woocommerce/ currently points to https://docs.dojo.tech/plugins/woocommerce/woocommerce/*. Note the second woocommerce/. I think it's a typo cause they all return 404.

**Comment by martinhay1** [comment]
Hi @malaporpism 
Thanks for flagging. This has now been fixed 👍
